.evergreen/atlas/teardown.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Handle proper teardown of Atlas.
+
+set -o errexit  # Exit the script with error if any of the commands fail
```

Comment:
imo this documentation comment is unnecessary: it provides no additional context or information beyond what is immediately apparant ("yes, that is exactly what `errexit` does"). Suggest its removal from this + similar scripts.

Reply:
Removed